boot/cmdline.go

```diff
@@ -169,7 +175,6 @@ func composeCommandLine(model *asserts.Model, currentOrCandidate int, mode, syst
 		}
 		return "", err
 	}
-	extraArgs := ""
 	if gadgetDirOrSnapPath != "" {
 		sf, err := snapfile.Open(gadgetDirOrSnapPath)
```

Comment:
my old comment about Open and the gadget helper remains

Reply:
Yes, it's in my TODO list

Reply:
Opened #10161 looks like it shouldn't conflict with the branches that are open for reviews now.